android-components/components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlinx/coroutines/flow/Flow.kt

```diff
@@ -9,46 +9,6 @@ import kotlinx.coroutines.flow.asFlow
 import kotlinx.coroutines.flow.filter
 import kotlinx.coroutines.flow.flatMapConcat
 
-/**
```

Comment:
I like the examples on these funcs, shame the Kotlin docs don't do the same

Reply:
Checkout https://flowmarbles.com/#distinctUntilChanged